### DISCUSSION TITLE
XQuartz on M1 Apple

### DISCUSSION THREAD

**Discussion by m77it** [XQuartz/XQuartz#381] [Q&A] [answered]
Hi all,
I installed Xquartz but it starts jumping and never works. Furthermore, the laptop is frozen since Xquartz tries to start and cannot be killed, so I need to restart my laptop.
I'm using Apple Silicon on Sonoma.
Any suggestion?
Thanks
Marco

**Comment by yue2022** [comment]
"I am experiencing the same problem as you. Have you discovered any solutions for it? Thank you."

**Reply by m77it** [reply]
No, I simply removed XQuartz

**Comment by jeremyhu** [comment, marked as answer]
This might be a delay while the font cache is being generated.  Yeah, it would be better to have that status communicated to users rather than just bouncing the dock icon =/
